src/alpaca_typer.erl

```diff
@@ -143,7 +143,7 @@ copy_cell(Cell, RefMap) ->
             {RevMembers, Map2} = lists:foldl(F, {[], RefMap}, Members),
             {RV2, Map3} = copy_cell(RV, Map2),
             {new_cell(#t_record{members=lists:reverse(RevMembers), row_var=RV2}), Map3};
-        #adt{vars=TypeVars, members=Members}=ADT ->
+        #adt{vars=TypeVars, members=Members, module=Mod}=ADT ->
```

Comment:
I think there are a couple of warnings coming from leex and yecc that might be tricky to squash but I could be wrong.  We could certainly try it.

Regardless, thanks for pointing this out.